ui/StatusQ/src/stringutilsinternal.cpp

```diff
@@ -10,11 +16,21 @@ QString StringUtilsInternal::escapeHtml(const QString &unsafe) const
     return unsafe.toHtmlEscaped();
 }
 
-QObject* StringUtilsInternal::qmlInstance(QQmlEngine *engine,
-                                          QJSEngine *scriptEngine)
+QString StringUtilsInternal::readTextFile(const QString& filePath) const
```

Comment:
I wonder if we could somehow indicate in the method name a bit more precisely what's doing (I meant resolving name via selector).
Maybe `readLocalizedTextFile` or `readTextAssetFile`, wdyt?

Reply:
That's an implementation detail imho; turned the other way around, if you supply an absolute path (`/home/foo/bar.txt`) it will still work, no "localized" or "assets" happening here

Reply:
Yeah, I understand, but in other cases the `localization` will happen, what's not expected from method called just `readTextFile` imo. But it's a detail. Go ahead with current name if you prefer.